HW13/HW13.cpp

```diff
@@ -0,0 +1,205 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <cstdlib>
+#include <cctype>
+
+int dayOfYear = 1;
+int currentData = 1;
+bool currentDay = false;
```

Comment:
mark it somehow to make it clear they are global. e.g. with `g_` prefix, or all caps, or similar